components/portal/src/img/githubLogo.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
```

Comment:
Please update the file names of the Image Components to have capital letters as well.